lib/segment/open.c

```diff
@@ -13,8 +13,12 @@
 
 #include <unistd.h>
 #include <fcntl.h>
+#include <string.h>
+#include <errno.h>
+
 #include <grass/gis.h>
 #include <grass/glocale.h>
+
```

Comment:
Why is this file only changing the includes? Were there some changes before, but no real ones after updating the PR with main?

Reply:
My bad. As I performed a lot of changes manually, I have messed up at some point :(